src/get-started/svgs/index.njk

```diff
@@ -0,0 +1,20 @@
+---
+title: Exporting SVGs
+sortOrder: 3
+---
+
+When exporting SVG images for the web you should ensure that styles are exported as *inline* rather than as CSS. This is to ensure that styles are not accidentally overridens as the generated class names are not unique, and allows [SVGO](https://github.com/svg/svgo) to optimise the file without breaking it.
```

Comment:
```suggestion
When exporting SVG images for the web you should ensure that styles are exported as *inline* rather than as CSS. This is to ensure that styles are not accidentally overridden as the generated class names are not unique, and allows [SVGO](https://github.com/svg/svgo) to optimise the file without breaking it.
```